README.md

```diff
@@ -148,42 +148,33 @@ These images contain only the package registry, they don't contain any package.
 
 ### Testing with Kibana
 
-The Docker image of Package Registry is just an empty distribution without any packages. To test it with Kibana using
-[elastic-package](https://github.com/elastic/elastic-package), you need to rebuild the snapshot distribution first:
+The Docker image of Package Registry is just an empty distribution without any packages.
+By default, Package Registry has the proxy mode enabled. Thereby, even if there is no package in the
```

Comment:
I think that proxy mode is not enabled by default. It is only in `elastic-package`, with `EPR_FEATURE_PROXY_MODE=true`.
We could tough mention how to enable it here.

Reply:
I've added a new section for the proxy mode feature in the Readme